discrete_math/euclidean_algorithm/EuclidianAlgo/euclidian.hpp

```diff
@@ -0,0 +1,155 @@
+// USE C++20 MODULES
+// READ e.g. https://learn.microsoft.com/en-us/cpp/cpp/modules-cpp?view=msvc-170
+
+//Podpinanie bibliotek
+#include <concepts>
+#include <tuple>
+#include <string>
+#include <algorithm>
+#include <numeric>
+#include <ranges>
+#include <iostream>
+
+namespace algorithms 
+{
+	template <std::integral... Is>
+	struct euclidean_result //Struktura zawierająca współczynniki oraz GCD(NWD)
+	{ 
+		std::tuple<Is...> coefficients{}; //Współczynniki
+		std::uint64_t GCD{}; //GDC
+	};
+
+	template <std::integral I, std::integral J>
+	constexpr auto gcd_extended(I i, J j) -> euclidean_result<I, J> //Funkcja obliczająca współczynniki oraz GCD dla 2 wartości

```

Comment:
remove inline comments, documentation for a function should appear just before the function